share/spice/cve_summary/cve_summary.js

```diff
@@ -0,0 +1,41 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_cve_summary = function(api_result){
+
+        // fail the spice if the API does not return a result
+        if(api_result == undefined) {
```

Comment:
@MoriTanosuke it looks like they actually return an empty object, not an undefined object.

You'll probably want to use http://api.jquery.com/jQuery.isEmptyObject/ to verify the object isn't empty!